assets/root-files/phpunit.xml.dist

```diff
@@ -1,7 +1,7 @@
-<phpunit bootstrap="./test/Bootstrap.php" colors="true">
+<phpunit bootstrap="./Bootstrap.php" colors="true">
```

Comment:
No; the phpunit.xml.dist file is in the root, and the test bootstrap is under `test/`. The original was correct.